website/docs/cloud-docs/run/cli.mdx

```diff
@@ -101,7 +101,7 @@ Remote runs in Terraform Cloud use:
 
 - Run-specific variables set through the command line or in your local environment. Terraform can use shell environment variables prefixed with `TF_VAR_` as input variables for the run, but you must still set all required environment variables, like provider credentials, inside the workspace.
 - Workspace-specific Terraform and environment variables set in the workspace.
-- Variable sets applied to the workspace.
+- Variable sets applied globally, to the project containing the workspace, or to the workspace.
```

Comment:
Feel free to disregard this one if you don't like it! I just like the extra specificity:
```suggestion
- Any variable sets applied globally, on the project containing the workspace, or on the workspace itself.
```